src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -118,12 +128,28 @@ getPendingData = do
 forModuleNameFix :: (Bool -> BlockHandler logger e a) -> BlockHandler logger e a
 forModuleNameFix f = use bsModuleNameFix >>= f
 
+-- TODO: speed this up, cache it?
```

Comment:
Two possibilities:
- maybe `NewBlock` time is delayed because this is slow. we should avoid using `rewoundPactDb` in most cases when we do `NewBlock` but rocksdb is slightly lagging the checkpointer so this happens more often than it should. 
- maybe read-only replays are slower than they could be because this is slow.

I'm fine with allocating a ticket to fixing those issues, but doing a cache is a bit heavy-duty. I think we get 99% of the way there by fixing the rocksdb race for `NewBlock` and hardcoding `coin_coin-table` into `knownTables` for read-only replay.